serverless-logic/serverless-logic-managing-services/serverless-logic-configuring-openAPI-services.adoc

```diff
@@ -0,0 +1,13 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="serverless-logic-configuring-openAPI-services_{context}"]
+= Configuring OpenAPI services
+:context: serverless-logic-configuring-openAPI-services
+include::_attributes/common-attributes.adoc[]
```

Comment:
Guidelines put the context line below the include for attributes. Best follow this as the customer portal can be funky.

Reply:
Okay